### PR TITLE
Vlang implementation

### PR DESCRIPTION
**Pull request by marghidanu**
## Description

Vlang implementation.

## Contributing requirements

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct category folder.
* [x] I added a README.md or Dockerfile.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeV/solution_1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM alpine:3.13 AS build
+
+RUN apk add --no-cache build-base bash git
+
+WORKDIR /opt/vlang
+RUN git clone https://github.com/vlang/v /opt/vlang && make
```
Comment (rbergen) [inline PrimeV/solution_1/Dockerfile:6]: Nice. It's been a while since I've seen the toolchain for a language been built from a `git clone` 😄 👍

**Review by rbergen — APPROVED** [review]
LGTM. Nice to have such a "new" language in the set of solutions.

File: PrimeV/solution_1/Dockerfile (hunk 1 above)
Comment (marghidanu) [reply PrimeV/solution_1/Dockerfile:6]: Yes, that's how new the language is! :)

**Comment by Ekopalypse** [comment]
@marghidanu - I have [here](https://github.com/Ekopalypse/V-examples/blob/main/primes.v) an optimized version of your V solution. Could you review it?
I can hardly add it as solution2, because it is basically your solution with small changes, but they have an impact.
Thank you.

**Comment by rbergen** [comment]
@Ekopalypse Just to confirn: I understand correctly that, you're not proposing to apply your changes to the Vlang solution that's currently in this repo?

**Comment by Ekopalypse** [comment]
@rbergen 
Sorry for the confusion, I am not a native speaker. No, I want it to be applied. But I think I can't do PR as solution2 that is basically the same as solution1 only with a few performance improvements.

**Comment by rbergen** [comment]
@Ekopalypse  In that case I recommend you open a PR not to add a second solution, but to improve the existing solution_1. Do note that you should only include those changes that actually improve the performance of the benchmarked code.

**Comment by Ekopalypse** [comment]
@rbergen I see - thank you very much for your help.

**Pull request merged**